projects/ngx-formentry/src/form-entry/form-renderer/form-renderer.component.html

```diff
@@ -289,8 +298,8 @@
             [minlength]="node.question.extras.questionOptions.minLength"
             [formControlName]="node.question.key"
             [attr.placeholder]="node.question.placeholder | translate"
-          >
-          </ofe-number-input>
+            [invalid]="node.control.touched && !node.control.valid"
```

Comment:
I'm basing the invalid validation state on whether the control has been touched and its value is invalid.